website/snippets/_new-sl-setup.md

```diff
@@ -0,0 +1,33 @@
+You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+- You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must have a successful run in your new environment.
+
+:::tip
+If you're using the legacy Semantic Layer, we **highly** recommend you [upgrade your dbt version](/docs/dbt-versions/upgrade-core-in-cloud) to dbt v1.6 or higher to use the new dbt Semantic Layer. Refer to the dedicated [migration guide](/guides/migration/sl-migration) for more info.
+:::
+
+1. in dbt Cloud, create a new [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) or use an existing environment on dbt 1.6 or higher.
+    * Note &mdash; Deployment environment is currently supported (_development experience coming soon_)
+
+2. Navigate to **Account Settings** and select the specific project you want to enable the Semantic Layer for.
+
+3. In the **Project Details** page, navigate to the **Semantic Layer** section, and select **Configure Semantic Layer**.
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-initial-page.jpg" width="60%" title="Semantic Layer section in the Project Details page"/>
+
+4. In the **Set Up Semantic Layer Configuration** page, enter the credentials you want the Semantic Layer to use specific to your data platform. 
+    * Note: We recommend using a less privileged set of credentials because your Semantic Layer users will be querying it in downstream applications. At a minimum, the Semantic Layer needs to have read access to the schema(s) that contains the dbt models that you used to build your semantic models.
+
+5. Select the deployment environment you want for the Semantic Layer and click **Save**.
```

Comment:
I was going to edit this, but now I'm not sure what I'd change. I find it confusing that we call them deployment environments and it also has a "Set as production environment" toggle and it's named "Production." Maybe it's me?

Reply:
i think we use deployment more and its what we use in the UI. but also use production a lot too. in this case, the screenshot example displays a deployment environment named 'Production.' It's usually best practice to name your deployment environment 'production' or 'prod' and your development environment 'dev' or 'development'

Reply:
i updated the screenshot title so it's clear that this is what the deployment env is named as:

>To set up your Semantic Layer configuration, enter your credentials and select the deployment environment. This example selects a deployment environment named 'Production.'

![Screenshot 2023-07-27 at 10 18 17](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/a33f74e6-064b-422b-8b30-f342a851fb1c)